lib/benchee/formatters/unit.ex

```diff
@@ -28,21 +35,27 @@ defmodule Benchee.Unit do
 
   @callback magnitude(unit) :: number
 
-  def float_precision(float) when float < 0.01, do: 5
-  def float_precision(float) when float < 0.1, do: 4
-  def float_precision(float) when float < 0.2, do: 3
-  def float_precision(_float), do: 2
+  @doc """
+  A separator that should appear between a value and a unit label. For example,
+  a space: `5.67 M` or an empty string: `5.67M`
+  """
+  @callback separator :: String.t
```

Comment:
The separator always seems to be `" "` atm. What's the rationale behind making it dependent on the unit?